recipes/msgf_plus/msgf_plus.py

```diff
@@ -71,7 +71,7 @@ def main():
     jar_dir = real_dirname(sys.argv[0])
     (mem_opts, prop_opts, pass_args) = jvm_opts(sys.argv[1:])
 
-    if pass_args != [] and pass_args[0].startswith('eu'):
+    if pass_args != [] and pass_args[0].startswith('edu'):
```

Comment:
Only edu, MSGFplus jar contains two executables that have to be run with `-cp` instead of `-jar`. Both of them are called `edu.ucsd.something_more`.